x-pack/metricbeat/module/syncgateway/_meta/testdata/expvar.282c.json

```diff
@@ -0,0 +1,1224 @@
+{
+    "cb": {
+        "ops": {},
+        "pools": {}
+    },
+    "cmdline": [
+        "sync_gateway",
+        "--defaultLogFilePath=/var/log/sync_gateway",
+        "-adminInterface",
+        ":4985",
+        "/etc/sync_gateway/sync_gateway.json"
+    ],
+    "goblip": {},
+    "mc": {
+        "recv": {
+            "bytes": {
+                "0x1f": 1008,
+                "0x89": 864,
+                "SASL_AUTH": 864,
+                "SASL_LIST_MECHS": 2376,
+                "UPR_OPEN": 864,
+                "total": 5976
+            },
+            "errs": {},
```

Comment:
should we ignore the field when it's empty?

Reply:
We don't actually parse it anyways, so it's effectively ignored 100% of times 😄 Anyways, be aware that this is not the `data.json` file but the response from the service, just in case 😬

Reply:
Ahh sorry I thought this is the data.json 😃